core/commands/block.go

```diff
@@ -249,13 +249,13 @@ It takes a list of base58 encoded multihashes to remove.
 
 			err = api.Block().Rm(req.Context, rp, options.Block.Force(force))
 			if err != nil {
-				err := res.Emit(&util.RemovedBlock{
+				if err := res.Emit(&util.RemovedBlock{
 					Hash:  rp.Cid().String(),
 					Error: err.Error(),
-				})
-				if err != nil {
+				}); err != nil {
```

Comment:
@Stebalien may disagree with me, but I would keep the original syntax.  I had to study this to figure out what is going on since the function call was not on a single line and there are a lot of extra parenthesis involved.

Reply:
I really don't care one way or another. To me, both are clear.